src/crypto/CryptoHash.h

```diff
@@ -27,6 +27,7 @@ class CryptoHash
 public:
     enum Algorithm
     {
+        Sha1,
```

Comment:
This change assumes nothing depends on the order or numeric values of the enum elements. It does not look like anything does, but I wanted to highlight this here anyway just in case.